package.json

```diff
@@ -7,7 +7,7 @@
     "start": "node ./bin/www",
     "e2e-test": "mocha test/e2e --timeout 15000",
     "test": "jasmine-node test/spec/ --forceexit",
-    "test:nsp": "echo 'TODO Implement NSP'",
+    "test:nsp": "nsp check",
```

Comment:
if test:nsp passes, perhaps we can include this in .travis.yml

Reply:
I thought about this but, for some reason, CMC don't do this - even though they do run `nsp check`